packages/forklift-console-plugin/src/modules/Plans/utils/helpers/getConditionTypes.ts

```diff
@@ -0,0 +1,17 @@
+import { V1beta1Plan } from '@kubev2v/types';
+
+import { PlanConditionStatus } from '../types/PlanCondition';
+
+/**
+ * Gets a record of plan types with truthful ('True') statuses
+ * @param plan V1beta1Plan
+ * @returns Record<string, boolean>
+ */
+export const getConditionTypes = (plan: V1beta1Plan): Record<string, boolean> =>
```

Comment:
Type string can be type PlanConditionType? will be more accurate and maintainable

Reply:
I thought about this as well, but it would impact this reduce like so:
<img width="804" alt="image" src="https://github.com/user-attachments/assets/7685d509-0c36-47c0-b967-e8c852f644f6" />